### ISSUE TITLE
rtl8831bu support

### ISSUE THREAD

**Issue by maurerr** [morrownr/rtl8852bu#32]
hi,

Thank you very much for your efforts supporting these drivers !
any chance you could add support for rtl8831bu devices (few dollars on AE) ?
I've successfully installed on ubuntu this newer (1.19.10) driver https://github.com/heesn/rtl8831

cheers !

**Comment by morrownr** [comment]
Hi @maurerr 

Sorry for the delay. I have been busy.

> any chance you could add support for rtl8831bu devices (few dollars on AE) ?

I won't rule out setting up a repo for rtl8851/31bu but it won't be something that can happen until at least next year. My enthusiasm for Realtek's WiFi 6 out of kernel drivers is not there and I have other projects that should help Linux users more in the long run.

Here is a message I posted recently:

https://github.com/morrownr/USB-WiFi/issues/449

I know it is not something you want to hear but I step back to look at the big picture at times and I came to the conclusion earlier this year that I needed to do my best to WARN Linux users to stay away from Realtek WiFi 6 usb adapters as best I could. Why? These Realtek WiFi 6 out-of-kernel drivers are not only non-standards compliant but they are just not very good at all. It is highly likely that we will see a drop off in community support for these drivers for a variety of reasons. In fact, I think it has been in progress for a while now. 

Who wants to work on something that is dead end? Kernel management made decisions about wifi many years ago as we moved into the world of WiFi 5. There is a Linux Wireless Standard that has to be followed. Realtek has not followed the standard. Mediatek has.

The Main Menu for this site is:

https://github.com/morrownr/USB-WiFi

The Main Menu has a lot of useful information. The first 3 menu items would be a good start. There are many really good adapters that work very well on Linux. You do not have to hunt and compile drivers. Seriously, pick yourself a good mt7921au based adapter from the Plug and Play List and give it a try. The Edup adapter that is listed first is a low cost option. Linux support for this adapter is constant and will probably last 20+ years. Until such time as Realtek wakes up, it really is time to avoid usb adapters based on their chips.

Regards,

@morrownr
